client/me/help/help-contact/index.jsx

```diff
@@ -694,6 +683,8 @@ export default connect(
 			ticketSupportConfigurationReady: isTicketSupportConfigurationReady( state ),
 			ticketSupportEligible: isTicketSupportEligible( state ),
 			ticketSupportRequestError: getTicketSupportRequestError( state ),
+			hasMoreThanOneSite: get( getCurrentUser( state ), 'visible_site_count', 0 ) > 0,
```

Comment:
The one who best replicates the old check is the getCurrentUserSiteCount. Although the sites-selector component just shows visible sites, so I thought if the user had 2 invisible sites we would be asking what site to select without allowing the user to select any site so I used the visible sites.
But after your comment I rechecked the things and sites-selector allows to select invisible sites if you search for them, so I correct to use  getCurrentUserSiteCount. 👍 Thank you for noticing this!